lib/player.js

```diff
@@ -1456,6 +1456,10 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           this.config_.manifest.retryParameters,
           has.mimeType);
     }
+    // Don't read video segments if the player is attached to an audio element
+    if (wants.mediaElement && wants.mediaElement.nodeName === 'AUDIO') {
+      this.config_.manifest.disableVideo = true;
```

Comment:
You'll want to make a local copy of `this.config_.manifest` before modifying it. Then you can pass that local copy in to `this.parser_.configure`.
You'll also want to apply the same local transformation inside `shaka.Player.applyConfig_`, so that it won't be forgotten if the user changes a configuration value during playback.
As-is, the video will continue to be disabled even if you re-attach to a video element. And honestly, the player changing the configuration permanently under your nose is kind of bad anyway... it's just going to lead to confusion.

Reply:
I made the changes. Can you review again?

Reply:
Yeah, looks good!